File: JavaScript/2148_count_elements_with_greater_and_smaller/solution.js
function count_elements(nums) {
    let result = new Array();
    for (let index = 0; index < nums.length; index ++) {
        if (Math.min(...nums) < nums[index] && Math.max(...nums) > nums[index]) {
            result.push(nums[index]);
        }
    }
    return result.length;
}

const nums = [-3,3,3,90];
console.log(count_elements(nums));

File: Python/2148_count_elements_with_greater_and_smaller/solution.py
def count_elements(nums):
    result = list()
    for index in range(len(nums)):
        if min(nums) < nums[index] and max(nums) > nums[index]:
            result.append(nums[index])
    return len(result)

nums = [-3,3,3,90]
print(count_elements(nums))